politeiawww/pi/mail.go

```diff
@@ -289,7 +289,7 @@ func (p *Pi) mailNtfnCommentReply(token string, commentID uint32, commentUsernam
 		return err
 	}
 
-	subject := "New Reply To Your Comment"
+	subject := "New Reply To Your Comment " + fmt.Sprint(commentID)
```

Comment:
Thought commentID is better option than record token here, as user might have multiple comments on the same proposals, maybe it makes sense to somehow combine both?

Reply:
I think it makes sense to have all the comment notifications for the same proposal in the same email thread.  You would use the token to do this.  The original issue was to fix the UX of notifications from different proposals appearing in the same email thread.

Reply:
Done.